readme.md

```diff
@@ -191,6 +191,7 @@
 	- [Protips](https://github.com/AllThingsSmitty/css-protips#readme)
 - [React](https://github.com/enaqx/awesome-react#readme) - App framework.
 	- [Relay](https://github.com/expede/awesome-relay#readme) - Framework for building data-driven React apps.
+	- [Design Systems](https://github.com/jbranchaud/awesome-react-design-systems#readme) - React-based Design Systems.
```

Comment:
> The entry in the Awesome list should:
Include a short description about the project/theme of the list. It should not describe the list itself.

The current description is just repeating the title. Instead, succinctly describe what a design system is.